README.md

```diff
@@ -0,0 +1,174 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <br/>
+  <h3> 🚀 <b>Tour Reservation</b> 🚀</h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [:camera: project screenshot](#screen-shoot)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🚀 Kaban Board](#Kaban-Board)
+    - [Kaban Board Initial State](#initial-state)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 🚀 Tour Reservation <a name="about-project"></a>
+
+<p> A tour reservation app is a software application designed to facilitate the process of booking and managing tour reservations. It allows users to browse available tours, select desired tour packages, specify booking details and receive confirmation for their reservations.</p>
+
+
+## Backend
+
+The backend was made with ruby on rails and can be found [here](https://github.com/Johnadibe/tour-reservation-app-api)
+
+
+
```

Comment:
[OPTIONAL] 
Consider adding a screenshot to give an idea of what app should look like. (not too many, 1 or 2 is fine)